channel-sdk/src/main/java/com/zello/channel/sdk/SessionConnectError.kt

```diff
@@ -90,6 +111,7 @@ class SessionConnectError @JvmOverloads constructor(val code: Code, val info: Ma
 		@JvmStatic internal val UNKNOWN = SessionConnectError(Code.UNKNOWN)
 		@JvmStatic internal val INVALID_ADDRESS = SessionConnectError(Code.INVALID_ADDRESS)
 		@JvmStatic internal val CONNECT_FAILED = SessionConnectError(Code.CONNECT_FAILED)
+		@Suppress("FunctionName")
```

Comment:
I don't think that works here because of the parameter. I've got no problem with the warning suppression. `BAD_RESPONSE` serves the same purpose as the other error constants, it just needs a way to specify its parameter, and the linter doesn't like that.

Reply:
You are right, my bad I missed the parameter